src/Symfony/Component/Security/Guard/Firewall/GuardAuthenticationListener.php

```diff
@@ -77,7 +77,12 @@ public function handle(GetResponseEvent $event)
             $this->executeGuardAuthenticator($uniqueGuardKey, $guardAuthenticator, $event);
 
             if ($event->hasResponse()) {
-                $this->logger->debug(sprintf('The "%s" authenticator set the response. Any later authenticator will not be called', get_class($guardAuthenticator)));
+                if (null !== $this->logger) {
+                    $this->logger->debug(sprintf(
+                        'The "%s" authenticator set the response. Any later authenticator will not be called',
+                        get_class($guardAuthenticator)
+                    ));
```

Comment:
@xabbuh even though it would be well over the PSR-2 recommended 120 character limit?